controller/test-files/search/show/assignee_null_negate.headers.golden.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
where is this file consumed? Or is it part of the setup and is used internally by tests?

Reply:
That is supposed to be in the last test inside the controller (`assignee=null with negate`). Will fix it.

Reply:
Fixed in 17f24ffbdbfd1a28ea5ee5deed34389a6a9ad015